mkosi/__init__.py

```diff
@@ -3717,35 +3711,41 @@ def make_verity_sig(
 
     assert root_hash is not None
 
-    try:
-        from OpenSSL import crypto
-    except ImportError:
-        die("Python OpenSSL package is not installed.");
+    from cryptography import x509
+    from cryptography.hazmat.primitives import hashes
+    from cryptography.hazmat.primitives import serialization
+    from cryptography.hazmat.primitives.serialization import pkcs7
```

Comment:
This is what `isort` will want to have, I didn't check that before
```suggestion
    from cryptography.hazmat.primitives import hashes, serialization
    from cryptography.hazmat.primitives.serialization import pkcs7
    from cryptography import x509
```